lms/static/scripts/frontend_apps/components/YouTubePicker.tsx

```diff
@@ -60,7 +70,7 @@ export default function YouTubePicker({
         </Button>,
         <Button
           data-testid="select-button"
-          disabled={!videoId}
+          disabled={!videoId || !!error || videoInfo.isLoading}
```

Comment:
Could this be written as `!videoInfo.data` instead?

Reply:
Yes, that works.